js/renovation/ui/check_box.tsx

```diff
@@ -83,6 +85,18 @@ export const viewFunction = (viewModel: CheckBox): JSX.Element => {
       </div>
       {viewModel.props.useInkRipple
                 && <InkRipple config={inkRippleConfig} ref={viewModel.inkRippleRef} />}
+      {viewModel.rendered && !viewModel.props.isValid && viewModel.props.validationStatus === 'invalid' && viewModel.validationErrors?.length
```

Comment:
viewFunction only for markup not for logic. I suggest to move
```
viewModel.rendered && !viewModel.props.isValid && viewModel.props.validationStatus === 'invalid' && viewModel.validationErrors?.length
```
into the viewModel. It's allow to simplify view tests

Reply:
fixed